tests/Console/Command/DescribeCommandTest.php

```diff
@@ -73,11 +73,11 @@ public function testExecuteOutputWithDecoration()
 Fixes stuff.
 Replaces bad stuff with good stuff.
 
-Fixer is configurable using following options:
+Fixer is configurable using following option:
```

Comment:
I've update the tests to test a whole bunch of stuff previously not tested.
There are still a lot of cases not tested, but I would like to move on with this PR and not try to cover all things that were not covered previously, I think that can be done in another PR.